### DISCUSSION TITLE
Will you provide a rust version?

### DISCUSSION THREAD

**Discussion by hiql** [darrenburns/posting#52] [Q&A]
Rust is very good to write cli apps, safe and fast!

example: gitui

**Comment by boldandbrad** [comment]
Are there specific technical challenges or shortcomings in the current tool that this would actually address or features that it would make possible?

I recognize that python as an install dependency isn't attractive to everyone for standalone tools, but I can't imagine the lift of a complete rewrite is worth it - other than for academic purposes. ...just my 2 cents.
